packages/blade/src/_helpers/storybook/StoryPageWrapper.tsx

```diff
@@ -0,0 +1,62 @@
+import {
```

Comment:
This is the newly added component.

Rest of the changes are basically removing the repetitive code and using this component to wrap the usage